config/routes.rb

```diff
@@ -3,10 +3,12 @@
   root "homes#index"
   get "/monsters/new", to: "homes#index"
   get "/monsters/:id", to: "homes#index"
+  get "/monsters/:id/review/new", to: "homes#index"
 
   namespace :api do
     namespace :v1 do
       resources :monsters, only: [:index, :show, :create]
+      resources :reviews, only: [:create]
```

Comment:
This `create` should be nested under `monsters`, because a review can only be created if it's associated with a monster. This also means that the path would change from
```
api/v1/reviews
```
to a more preferred
```
api/v1/monsters/:monster_id/reviews
```
giving us access to the `id` of the monster the review is for